docs/rules/no-important.md

```diff
@@ -0,0 +1,21 @@
+# no-important
+
+Disallow `!important` annotations.
+
+## Background
+
+Needing !important indicates there may be a larger underlying issue.
```

Comment:
We need more of a description here. :smile: Please see other rule docs for examples.